app/initializers/error.js

```diff
@@ -0,0 +1,61 @@
+import Ember from 'ember';
+import config from '../config/environment';
+
+const { $ } = Ember;
```

Comment:
This isn't a good idea since there is already a global $ defined and I've had problems in the past with doing this.  Either use Ember.$ or else don't use the variable $.